src/widgets/mentoring/ui/mentoring.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { screen } from '@testing-library/react';
+import { beforeEach, describe, expect, it } from 'vitest';
+import { renderWithRouter } from '@/shared/__tests__/utils';
+import { Mentoring } from '@/widgets/mentoring';
+import { mentorshipBenefits } from 'data';
+
+describe('Mentoring', () => {
+  beforeEach(() => {
+    renderWithRouter(<Mentoring />);
+  });
+
+  it('renders the title', () => {
+    expect(screen.getByText('Mentoring is for you if you')).toBeVisible();
+  });
+
+  it.each(mentorshipBenefits)('renders the benefit $id with the given info', ({ info }) => {
+    const benefit = screen.getByText(info);
+
+    expect(benefit).toHaveTextContent(info);
```

Comment:
I suppose this test-case doesn't make sense because you first search for an element by specific text and then check that this text matches the text you searched for. You can use `data-testid` for search benefit-items

Reply:
this actually worked as it should, but anyway i have changed the test